codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -6,7 +6,7 @@ import 'dart:async';
 import 'package:file/file.dart';
 import 'package:process/process.dart';
 import 'package:logging/logging.dart';
-import 'package:codesign/src/utils.dart';
+import 'utils.dart';
```

Comment:
https://dart-lang.github.io/linter/lints/directives_ordering.html

Reply:
True, I expanded the rules explicitly